spring-cloud-skipper-server-core/src/main/resources/application.yml

```diff
@@ -41,6 +41,7 @@ server:
 spring:
   main:
     banner-mode: "off"
+    allow-circular-references: true
```

Comment:
We can't do this. We need to fix the underlying cause of bean cycles as boot 2.6.x is now enforcing it.

Reply:
Found same flag in DataFlow and created a reproducer for spring-cloud-common-security-config.